src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnector.java

```diff
@@ -397,6 +447,32 @@ private Zone getZoneByTag(String zone) throws VCertException {
     return searchCertificates(Cloud.SearchRequest.findByFingerPrint(cleanFingerprint));
   }
 
+  private String[] parseZoneIdentifiers(String zone) throws VCertException {
+    try {
+      // Check if zone is UUID
+      UUID.fromString(zone);
+      return new String[] {zone, null, null};
+    } catch (IllegalArgumentException iae) {
+      // The zone argument is not UUID, so we expect to be ProjectName\ZoneName
+      String zoneParsed[] = zone.split(Pattern.quote("\\"));
```

Comment:
zones with `\\` in cloud? if this is common function why it in `cloud` directory?

Reply:
@mr-tron I think that is a valid alternate method for Venafi Cloud that we don't yet support in other VCert implementations.  I'll slack you the story about it in the backlog.